cmd/query/app/apiv3/package_test.go

```diff
@@ -6,9 +6,67 @@ package apiv3
 import (
```

Comment:
tests are always named after the file they are testing: `otlp_translator_test.go`